src/login/logind-dbus.c

```diff
@@ -1153,41 +1153,54 @@ static int manager_create_session_by_bus(
                                                  "No seat '%s' known", cseat);
         }
 
+        _cleanup_free_ char *resolved = NULL;
+
         if (isempty(tty))
                 tty = NULL;
-        else if (tty_is_vc(tty)) {
-                int v;
+        else {
+                if (tty_is_console(tty)) {
+                        r = resolve_dev_console(&resolved);
```

Comment:
> > Hmm, how does /dev/console mtime work? I wonder if this would have an impact on the tty idle logic
>
> urks, this is messy indeed. Yikes.

This is not yet addressed? Your use case should be covered by resolution done in pid1 IIUC, so let's not duplicate this in logind, to avoid unnecessarily altering the semantics. We can revisit in the future if it becomes necessary for other cases